File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int count = 1;
        int ans = 0;
        for(int i = 0; i<sentences.size(); i++){
            for(int j = 0; j<sentences[i].size(); j++){
                if (sentences[i][j] == ' '){
                    count++;
                }
                ans = max(count, ans);
    
            }
            
            count = 1;
        }
    
    return ans ;
    }
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max = 0;
        for(int i = 0; i<sentences.length; i++){
            int count = 0;
            for(int j = 0; j<sentences[i].length(); j++){
                if(sentences[i].charAt(j) == ' '){
                    count++;
                }  
            }
            
            if(count > max){
                max = count;        
            }
            
        }
        
        return max + 1;
    }
}